.github/workflows/ci-code-review-rust.yml

```diff
@@ -41,20 +36,12 @@ jobs:
       - name: Checkout submodules
         run: git submodule update --init
 
-      - name: Install Linux dependencies
-        run: sudo apt-get update && sudo apt-get install -y pkg-config build-essential libudev-dev
-
-      - name: Install Rust
-        uses: actions-rs/toolchain@v1
-        with:
-          override: true
-          profile: minimal
-          toolchain: ${{ env.RUST_TOOLCHAIN }}
-          components: rustfmt, clippy
-
```

Comment:
@silas-x How does this work? We need an action to install the rust toolchain, right?